python-sdk/src/transfers/integrations/__init__.py

```diff
@@ -0,0 +1,27 @@
+import importlib
+
+from transfers.constants import IngestorSupported
+from transfers.integrations.base import TransferIntegrations
+
+from astro.utils.path import get_class_name
+
+CUSTOM_INGESTION_TYPE_TO_MODULE_PATH = {"Fivetran": "transfers.integrations.fivetran"}
+
+
+def create_transfer_integration(transfer_params: dict = {}) -> TransferIntegrations:
+    """
+    Given a transfer_params return the associated TransferIntegrations class.
+
+    :param transfer_params: kwargs to be used by methods involved in transfer using FiveTran.
+    """
+    ingestion_type_name = transfer_params.get("ingestion_type", None)
+    if ingestion_type_name is None:
+        raise ValueError("ingestion_type not specified.")
+    if ingestion_type_name not in {item.value for item in IngestorSupported}:
+        raise ValueError("Ingestion platform not yet supported.")
+
+    module_path = CUSTOM_INGESTION_TYPE_TO_MODULE_PATH[ingestion_type_name]
+    module = importlib.import_module(module_path)
+    class_name = get_class_name(module_ref=module, suffix="Integration")
+    transfer_integrations: TransferIntegrations = getattr(module, class_name)(transfer_params)
```

Comment:
Do we really have to dynamically import? I think for simplicity, we should map type to class directly. E.g.
```python
from transfers.integrations.fivetran import FivetranIntegration

CUSTOM_INGESTION_TYPE_TO_MODULE_PATH = {"Fivetran": FivetranIntegration}
```

Because if you really want to do all of this dynamically you need to take care of more things. For example, `suffix="Integration"` is very hidden whereas the definition of the class(name) is in one place i.e. `class FivetranIntegration`. One could easily forget that the class has to have an integration suffix.

Note also that your exceptions on the module path only fail on run-time - not during parsing.

I think as long as a dynamic import is not necessary, I would go with static imports instead.